packages/babel-types/src/modifications/typescript/removeTypeDuplicates.ts

```diff
@@ -41,7 +41,7 @@ export default function removeTypeDuplicates(
 
     if (isTSUnionType(node)) {
       if (typeGroups.indexOf(node.types) < 0) {
-        nodes = nodes.concat(node.types);
+        nodes.push(...node.types);
```

Comment:
Also here.

Reply:
In the TS version `nodes` is not marked as readonly, should we align it to flow or vice versa?

Reply:
Yeah, we can mark the TS version as readonly. In general, public APIs should treat their parameters as readonly unless the function name clearly indicates that it's mutating.